src/util/clamp.ts

```diff
@@ -0,0 +1,11 @@
+/**
```

Comment:
some files are properly renamed, but some are not like clamp. The diff shows a removal of .js and new .ts file. I would like to preserve the history by using a proper rename/move.

Reply:
I think I have to use git rebase to rewrite the history, will that break the pull request?